index.html

```diff
@@ -168,7 +165,7 @@ <h2 class="project-subtitle">RPM Governor</h2>
           <a href="#" class="button">See Project</a>
         </article>
```

Comment:
### Your buttons
![buttons](https://user-images.githubusercontent.com/93039359/178596071-2ecee23e-c86d-4f7d-b2dd-609440b85add.png)

- [ ] Your buttons are lacking the 4.5:1 contrast. Please fix it :+1:

Reply:
Solved, changing the color of the buttons to a darker one. The test now show 4.86